DOF2DMD/settings.ini

```
@@ -1,21 +1,23 @@
-; Settings for DOF2DMD
-; The base URL that DOF2DMD will listen to. Defaults to http://127.0.0.1:8080
-url_prefix=http://127.0.0.1:8080
-; Display the score for x seconds, then back to marquee. Defaults to 5 seconds.
-;display_score_duration_s=5
-; Verbose output in debug.log file if debug=true. Defaults to false.
-;debug=false
-; Path of the artwork (relative to DOF2DMD or absolute). Defaults to "artwork"
-;artwork_path=artwork
-; Width in pixels for DMD. Defaults to 128
-;dmd_width=128
-; Height in pixels for DMD. Defaults to 64
-;dmd_height=64
-; Picture to display when DOF2DMD starts. Defaults to DOF2DMD (that is artwork/DOF2DMD.png or DOF2DMD.gif)
-;start_picture=DOF2DMD
-; Not implemented ---
-;scene_default=marquee
-;number_of_dmd=1
-;animation_dmd=1
-;score_dmd=1
-;marquee_dmd=1
+; Settings for DOF2DMD
+; The base URL that DOF2DMD will listen to. Defaults to http://127.0.0.1:8080
+url_prefix=http://127.0.0.1:8080
+; Display the score for x seconds, then back to marquee. Defaults to 5 seconds.
+;display_score_duration_s=5
+; Verbose output in debug.log file if debug=true. Defaults to false.
+;debug=false
+; Path of the artwork (relative to DOF2DMD or absolute). Defaults to "artwork"
+;artwork_path=artwork
+; Width in pixels for DMD. Defaults to 128
+dmd_width=256

```

Should be a comment, and keep 128 as default.
```suggestion
;dmd_width=256
```

---

Perfect for me (I'm testing in a 256x64) I forgot to left the default size